presets/requirement/README.md

```diff
@@ -0,0 +1,3 @@
+# Requirement
+
+Pip dependencies for both Kaito inference and tuning.
```

Comment:
For here, can we add some more instructions to clarify this is about vllm? From the path to the content I cannot see the scope of this Requirement

Reply:
it works for both vllm and transformers.